### PR TITLE
:memo: Create data validation FAQ

### PR DESCRIPTION
**Pull request by bpenteado**
See title.

### CODE REVIEW

File: docs/faq/data-validation.ipynb [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f6d385d2-08ed-438f-a61c-56ff7f12ab0a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "??lns.Pipeline"
```
Comment (falexwolf) [inline docs/faq/data-validation.ipynb:96]: This here prints nothing on the generated docs:
image

File: docs/faq/data-validation.ipynb [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "63125633-b71e-4b33-8001-8b752c93b144",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "??lns.User"
```
Comment (falexwolf) [inline docs/faq/data-validation.ipynb:57]: Prints no output on the generated docs!

**Comment by falexwolf** [comment]
Could we drop the term "complex type" everywhere and just refer to "relationships" instead?

"Complex type" sounds like a not-very-defined thing, and the only thing we're really interested in in this context are relationships.

**Comment by falexwolf** [comment]
The logging mentioning `created_at` at the very bottom seems to arise from a logging bug.

It should print `pipeline` instead of `created_at`, I assume!
image

image

**Comment by falexwolf** [comment]
Otherwise: It's a beautiful FAQ answering very relevant questions concisely!

**Comment by falexwolf** [comment]
Only title: I wouldn't refer to "LaminDB ORM", but rather say: "Data validation with ORMs".

(Capitalization should be lower case other than names and first letter of title, that's called "sentence case": https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings)

File: docs/faq/data-validation.ipynb (hunk 1 above)
Comment (bpenteado) [reply docs/faq/data-validation.ipynb]: Removed it!

File: docs/faq/data-validation.ipynb (hunk 2 above)
Comment (bpenteado) [reply docs/faq/data-validation.ipynb]: Removed it!

**Comment by bpenteado** [comment]
> Could we drop the term "complex type" everywhere and just refer to "relationships" instead?
> 
> "Complex type" sounds like a not-very-defined thing, and the only thing we're really interested in in this context are relationships.

Done. Will also change references in the work package.

**Pull request merged**